suite/tests/client-interface/drx-scattergather.dll.c

```diff
@@ -45,13 +45,73 @@
         }                                                                            \
     } while (0);
 
+static uint64 global_sg_count;
+
 static void
 event_exit(void)
 {
     drx_exit();
     drreg_exit();
     drmgr_exit();
-    dr_fprintf(STDERR, "event_exit\n");
+    dr_fprintf(STDERR, "event_exit, %d scatter/gather instructions\n", global_sg_count);
+}
+
+static void
+inscount(uint num_instrs)
+{
+    global_sg_count += num_instrs;
```

Comment:
Worth a comment that we're assuming the app is single-threaded and thus there's no race here.

Reply:
Done.